src/switch_rtp.c

```diff
@@ -4898,12 +4898,14 @@ SWITCH_DECLARE(switch_status_t) switch_rtp_set_remote_ssrc(switch_rtp_t *rtp_ses
 SWITCH_DECLARE(switch_status_t) switch_rtp_create(switch_rtp_t **new_rtp_session,
 												  switch_payload_t payload,
 												  uint32_t samples_per_interval,
-												  uint32_t ms_per_packet,
+												  uint32_t ms_per_pkt,
 												  switch_rtp_flag_t flags[SWITCH_RTP_FLAG_INVALID], char *timer_name, const char **err, switch_memory_pool_t *pool)
 {
 	switch_rtp_t *rtp_session = NULL;
 	switch_core_session_t *session = switch_core_memory_pool_get_data(pool, "__session");
 	switch_channel_t *channel = NULL;
+	uint32_t ms_per_packet = 20;
```

Comment:
We should initialize the value of this variable here instead initialize on line 4979
ms_per_packet = ms_per_pkt;
us_per_packet = 1000 * ms_per_packet;